Document anonymous access (#1023)
SAP/luigi: docs/navigation-parameters-reference.md

- **type**: boolean
- **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.

### anonymousAccess

> - **type**: boolean or "exclusive"

Reviewer: ```suggestion
- **type**: boolean or `exclusive`
```

> - **description**: when set to `true`, the node is always accessible. When set to `exclusive`, the node is only visible in logged-out state. Requires **auth.disableAutoLogin** to be set to `true`. **anonymousAccess** needs to be defined also on it's child nodes.

Reviewer: ```suggestion
- **description**: when set to `true`, the node is always accessible. When set to `exclusive`, the node is only visible in the logged-out state. Requires **auth.disableAutoLogin** to be set to `true`. **anonymousAccess** needs to be defined on both parent and child nodes.
```
